doc/source/cluster/kubernetes/user-guides/configuring-autoscaling.md

```diff
@@ -1,36 +1,18 @@
-(kuberay-autoscaler-discussion)=
-# Autoscaling
-This page discusses autoscaling in the context of Ray on Kubernetes.
-For details on autoscaler configuration see the {ref}`configuration guide<kuberay-autoscaling-config>`.
+(kuberay-autoscaling)=
 
-:::{note}
-Autoscaling is supported only with Ray versions at least
-as new  as 1.11.0.
-:::
+# Configuring Autoscaling with KubeRay
 
-(autoscaler-pro-con)=
-## Should I enable autoscaling?
-Ray Autoscaler support is optional.
-Here are some considerations to keep in mind when choosing whether to use autoscaling.
+This guide explains how to configure the Ray autoscaler on Kubernetes.
+The Ray autoscaler is a Ray cluster process that automatically scales a cluster up and down based on resource demand.
+The autoscaler does this by adjusting the number of nodes (Ray pods) in the cluster based on the resources required by tasks, actors or placement groups.
 
-### Autoscaling: Pros
-**Cope with unknown resource requirements.** If you don't know how much compute your Ray
-workload will require, autoscaling can adjust your Ray cluster to the right size.
+Note that the autoscaler only considers logical resource requests for scaling (i.e., those specified in ``@ray.remote`` and displayed in `ray status`), not physical machine utilization. If a user tries to launch an actor, task, or placement group but there are insufficient resources, the request will be queued. The autoscaler adds nodes to satisfy resource demands in this queue.
+The autoscaler also removes nodes after they become idle for some time.
```

Comment:
We could probably replace most instances of "node" with "Ray pod", to avoid confusing with Kubernetes nodes.